pkg/tmpl/context_funcs.go

```diff
@@ -69,6 +70,14 @@ func (c *Context) createFuncMap() template.FuncMap {
 		funcMap["readFile"] = func(string) (string, error) {
 			return "", nil
 		}
+		funcMap["readDir"] = func(string) ([]string, error) {
+			var paths []string
+			return paths, nil
```

Comment:
return nil, nil

Reply:
I thought of that but I see that readFile for returns empty string. I can change it but I am then I guess this should be changed for readFile or I am wrong?

Thanks